fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/YamlSectionHandler.java

```diff
@@ -3,27 +3,22 @@
 import ch.bbw.fabbwled.lands.book.SectionHandler;
 import ch.bbw.fabbwled.lands.book.SectionId;
 import ch.bbw.fabbwled.lands.book.SectionNode;
-import ch.bbw.fabbwled.lands.exception.FabledBusinessException;
+import ch.bbw.fabbwled.lands.character.PlayerDto;
 import ch.bbw.fabbwled.lands.service.PlayerSession;
 import lombok.RequiredArgsConstructor;
 
-import java.util.ArrayList;
-import java.util.function.Function;
-
 @RequiredArgsConstructor
 public class YamlSectionHandler implements SectionHandler {
     private final PlayerSession playerSession;
     private final YamlSection section;
 
-    private ArrayList<Function<PlayerSession.PlayerDto, PlayerSession.PlayerDto>> handlers = new ArrayList<>();
```

Comment:
In #556, I'm using the handlers for more fun things: https://github.com/keykey7/fabbwled/pull/556/files#diff-5e3224d8e0cd08c5df61abd75b431da8062e4c0cdd75c8a311a4d4eb90253890R93
That's gonna conflict, what should we do with those?

Reply:
merged